src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyCultureProvider.cs

```diff
@@ -32,7 +32,7 @@ internal WebAssemblyCultureProvider(CultureInfo initialCulture, CultureInfo init
     internal static void Initialize()
     {
         Instance = new WebAssemblyCultureProvider(
-            initialCulture: CultureInfo.CurrentCulture,
+            initialCulture: CultureInfo.GetCultureInfo(WebAssemblyCultureProviderInterop.GetApplicationCulture() ?? CultureInfo.InvariantCulture.Name),
             initialUICulture: CultureInfo.CurrentUICulture);
```

Comment:
I'm thinking if we shouldn't remove `initialUICulture` from this constructor. CIt's internal only, currently it's not used anywhere and is a relict of the misconception in previous PR that was changing the way we're loading the entry point. Is that correct?